### PR TITLE
7259 Added missing columns to schema doc

### PR DESCRIPTION
**Pull request by APriestman**

### CODE REVIEW

File: bindings/java/doxygen/schema/db_schema_8_6.dox [hunk 1]
```diff
@@ -62,8 +64,10 @@ Contains information about each set of images that is stored in the database.
 - **ssize** - Sector size of device in bytes
 - **tzone** - Timezone where image is from (the same format that TSK tools want as input)
 - **size** - Size of the original image (in bytes) 
-- **md5** - Hash of the image.  Currently, this is populated only if the input image is E01. 
-- **display_name** - display name of the image. 
+- **md5** - MD5 hash of the image
+- **sha1** - SHA-1 hash of the image
+- **sha256** - SHA-256 hash of the image
+- **display_name** - display name of the image
```
Comment (rcordovano) [inline bindings/java/doxygen/schema/db_schema_8_6.dox:70]: display => Display

File: bindings/java/doxygen/schema/db_schema_8_6.dox [hunk 2]
```diff
@@ -176,33 +184,37 @@ NOTE: This table is not used in any code.
...
+The \ref mod_bbpage "Blackboard" is used to store results from analysis modules. 
 
 ## blackboard_artifacts
 Stores artifacts associated with objects.
 - **artifact_id** - Id of the artifact (assigned by the database)
 - **obj_id** - Id of the associated object
 - **artifact_type_id** - Id for the type of artifact (can be looked up in the blackboard_artifact_types table)
+- **data_source_obj_id** - Id of the data source for this artifact
+- **artifact_type_id** - Type of artifact (references artifact_type_id in blackboard_artifact_types)
+- **review_status_id** - (references review_status_id in review_statuses)
```
Comment (rcordovano) [inline bindings/java/doxygen/schema/db_schema_8_6.dox:196]: How about we standardize on referring to "the artifact" and use that where we currently sometimes have "this artifact" or just "artifact." I don't actually care what we choose, but consistency would be nice.

File: bindings/java/doxygen/schema/db_schema_8_6.dox [hunk 3]
```diff
@@ -176,33 +184,37 @@ NOTE: This table is not used in any code.
...
 ## blackboard_attributes
-Stores name value pairs associated with an artifact. Only one of the value columns should be populated
+Stores name value pairs associated with an artifact. Only one of the value columns should be populated.
 - **artifact_id** - Id of the associated artifact.
+- **artifact_type_id** - Artifact type of the associated artifact.
 - **source** - Source string, should be module name that created the entry.
 - **context** - Additional context string
 - **attribute_type_id** - Id for the type of attribute (can be looked up in the blackboard_attribute_types)
-- **value_type** - The type of value (0 for string, 1 for int, 2 for long, 3 for double, 4 for byte array)
+- **value_type** - The type of value (see org.sleuthkit.datamodel.BlackboardAttribute.TSK_BLACKBOARD_ATTRIBUTE_VALUE_TYPE)
```
Comment (rcordovano) [inline bindings/java/doxygen/schema/db_schema_8_6.dox:205]: "of value" => "of the value"

**Review by rcordovano — CHANGES_REQUESTED** [review]

File: bindings/java/doxygen/schema/db_schema_8_6.dox (hunk 1 above)
Comment (APriestman) [reply bindings/java/doxygen/schema/db_schema_8_6.dox:70]: I am going to do a big cleanup PR to keep things like this separate from the actual changes, so please wait until then.

File: bindings/java/doxygen/schema/db_schema_8_6.dox (hunk 2 above)
Comment (APriestman) [reply bindings/java/doxygen/schema/db_schema_8_6.dox:196]: Let's also save this for a future PR

**Review by rcordovano — APPROVED** [review]

**Pull request merged**